### PR TITLE
bybit add safeTrade and feeCost to string for safeOrder2

### PR DESCRIPTION
**Pull request by DoctorSlimm**

### CODE REVIEW

**Comment by DoctorSlimm** [comment]
not sure what the error is, it builds on my laptop fine

**Comment by kroitor** [comment]
@DoctorSlimm it says:

> PHP Fatal error:  Uncaught ErrorException: Declaration of ccxt\bybit::fetch_order_trades($id, $symbol, $since = NULL, $limit = NULL, $params = Array) should be compatible with ccxt\Exchange::fetch_order_trades($id, $symbol = NULL, $params = Array) in /home/travis/build/ccxt/ccxt/php/bybit.php on line 2006 in /home/travis/build/ccxt/ccxt/php/test/syntax.php:13

**Comment by kroitor** [comment]
@DoctorSlimm despite that the symbol argument is required and the check for it is contained inside fetchOrderTrades, we still have to set the default value of `symbol` to `undefined` for it to match the other exchanges.

File: js/bybit.js [hunk 1]
```diff
@@ -2025,14 +2017,13 @@ module.exports = class bybit extends Exchange {
         };
         let market = undefined;
         if (symbol === undefined) {
+            throw new ArgumentsRequired (this.id + ' fetchMyTrades() requires a symbol argument');
```
Comment (kroitor) [inline js/bybit.js:2020]: You have tested and it is really required, right? If that is the case, we should move this check to the top of this method, cause if the `symbol` argument is missing but required, we should throw immediately.

**Pull request merged**